cloud/aws/services/ec2/tags.go

```diff
@@ -53,8 +61,8 @@ func (s *Service) clusterTagKey(clusterName string) string {
 }
 
 // createTags tags a resource with tags including the cluster tag
-func (s *Service) createTags(clusterName string, resourceID string, lifecycle ResourceLifecycle, additionalTags map[string]string) error {
-	tags := s.buildTags(clusterName, lifecycle, additionalTags)
+func (s *Service) createTags(clusterName string, resourceID string, lifecycle ResourceLifecycle, name string, role string, additionalTags map[string]string) error {
```

Comment:
can cluster the strings for a cleaner signature, optional, but `createTags(clusterName, resourceID, name, role string, ...)`